modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -74,15 +74,15 @@ spec:
     - file_mode=0777
   csi:
     driver: smb.csi.k8s.io
-    volumeHandle: smb-server.default.svc.cluster.local/share/ <2>
+    volumeHandle: smb-server.default.svc.cluster.local/share#subdir# <2>
     volumeAttributes:
       source: //<hostname>/<shares> <3>
     nodeStageSecretRef:
       name: <secret_name_shares> <4>
       namespace: <namespace> <5>
 ----
 <1> The name of the PV.
-<2> `volumeHandle` format: {smb-server-address}.{sub-dir-name}.{share-name}. Ensure that this value is unique for every share in the cluster.
+<2> `volumeHandle` format: {smb-server-address}#{sub-dir-name}#{share-name}. Ensure that this value is unique for every share in the cluster.
```

Comment:
@lpettyjo The variables appear in the preview. Let me know if I grabbed the wrong preview.
https://85928--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-smb-cifs.html